sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,408 @@
+            - Select CPU JTAG via strap pins.
+            - Load program into SRAM to execute OTP word write sequences.
+            - Trigger OTP writes to HW partition (e.g. device ID).
+            - Perform reset by toggling POR pin.
+            - Select CPU JTAG via strap pins.
+            - Verify that OTP write was successful.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_cp_device_info_flash_wr
+      desc: '''Verify that the tester is able to write to flash info pages.
+
+            During early manufacturing stages, the tester writes information for later use into
+            flash info pages.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Select CPU JTAG via strap pins.
+            - Load program into SRAM to execute flash write sequences.
+            - Write 32B to the isolated flash info partition. Verify unable to read back.
+
+            In silicon, a separate test program must perform the following additional operations.
+            In pre-silicon, they can be part of the same program.
+
+            - Switch to LC TAP at runtime.
+            - Request transition to DEV or PROD state (test both cases).
+            - Perform reset by toggling POR pin.
+            - Verify isolated flash info partition expected data.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_cp_test_lock
+      desc: '''Verify provisioning and use of unlock tokens.
+
+            In transit devices will need to be switched into TEST_LOCKED states to disable debug
+            access and code execution. This test verifies that a TEST_UNLOCKED token can be
+            provisioned as part of this process.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Select CPU JTAG via strap pins.
+            - Program TEST_UNLOCK_TOKEN in OTP transition via JTAG.
+            - Select LC TAP via strap pins (at runtime).
+            - Request transition into TEST_LOCKED state.
+            - Perform reset by toggling POR pin.
+            - Select LC TAP via strap pins.
+            - Verify lc state.
+
+            Implement the following steps in a way in which it could be executed at a later time
+            in silicon targets:
+
+            - Load OTP state generted by previous steps.
+            - Select LC TAP via strap pins.
+            - Request transition to TEST_UNLCOKED by providing the TEST_UNLOCK_TOKEN provisioned
+              in previous steps.
+            - Perform reset by toggling POR pin.
+            - Select LC TAP via strap pins.
+            - Verify lc_state.
+
+            Repeat the same flow for the number of supported TEST_LOCK/UNLOCKED states.
+            '''
```

Comment:
Notes to add:
cross-refernce: `chip_sw_rv_dm_lc_disabled`, `chip_sw_lc_walkthrough`, `chip_rv_dm_cpu_debug_mem_not_accessable`
- JTAG scan should not work.
- TAP commands should be unresponsive.
- CPU execution should be disabled.